src/libaudcore/playback.cc

```diff
@@ -429,6 +433,20 @@ static void * playback_thread (void *)
     return nullptr;
 }
 
+#ifdef USE_QT
+// QtMultiMedia is likely to use QObject timers (it does on Mac), which
+// need to be started from a thread created by QThread. So we use a thin
+// QThread wrapper instead of forking in the usual fashion with pthread.
+class PlaybackThread : public QThread
+{
+    void run() override
+    {
+        playback_thread(nullptr);
+        deleteLater();
+    }
+};
+#endif // USE_QT
```

Comment:
No.  If we need to stop using pthread, that's a bigger conversation and it should be done systematically, not piecemeal like this.

Reply:
More likely QtMultimedia needs to be fixed/dropped, though.